architecture/topics/auth_enabled_registry.adoc

```diff
@@ -0,0 +1,114 @@
+// This assembly is included in the following assemblies:
+//
+// <architecture/infrastructure_components/image_registry#auth-enabled-registry.adoc>
+//<install_config/registry/index#auth-enabled-registry.adoc>
+
+// This assembly can be included from other assemblies using the following
+// include statement:
+
+// The following line is necessary to allow assemblies be included in other
+// assemblies. It restores the `context` variable to its previous state.
+:parent-context-of-creating-build-inputs: {context}
+
+// The file name and the ID are based on the assembly title.
+// For example:
+// * file name: assembly_my-assembly-a.adoc
+// * ID: [id='assembly_my-assembly-a_{context}']
+// * Title: = My assembly A
+
+//Concept
+All container images available through the Red Hat Container Catalog are hosted
+on an image registry, `registry.access.redhat.com`. The Red Hat Container Catalog
+is moving from `registry.access.redhat.com` to `registry.redhat.io`.
+
+The new registry, `registry.redhat.io`, requires authentication for access to
+images and hosted content on {product-title}. Following the move to the new
+registry, the existing registry will be available for a period of time.
+
+[NOTE]
+====
+{product-title} pulls images from `registry.redhat.io`, so you will need to
+ensure your cluster is configured to access it.
+====
+
+The new registry uses standard OAuth mechanisms for authentication,
+with the following methods:
+
+* *Web username and password.* This is the standard set of credentials you use
+to log in to resources such as `access.redhat.com`.
+* *Authentication token.* These tokens, generated by administrators, are non-user
+accounts that give systems the ability to authenticate against
+the container registry.
+
+You can use `docker login` with your credentials, username and password or authentication
+token, to access content on the new registry.
+
+All image streams are updated to point to the new registry. Because the
+new registry requires authentication for access, there is a new secret in the
+OpenShift namespace called `imagestreamsecret`.
+
+There are two places your credentials must exist:
+
+* *OpenShift namespace*. Your credentials must exist in the OpenShift
+namespace so that the image streams in the OpenShift namespace can import.
```

Comment:
maybe s/so that the image streams in the OpenShift namespace can import./so that the images streams that the namepace references can import